File: CSIT111/Lecture_2/Lecture_Labs/Lecturer_Examples/Circle_2a.java
// Given a radius of a circle.
// - We wish to compute its area and its perimeter.
// - Display the info.
//
// File name: Circle_2a.java

class Circle_2a
{
	public static void main (String [ ] args)
	{
		// Declare a constant pi
		final double PI = 3.14159256;
		
		// Declare variables
		double radius;
		double area, perimeter;
		
		// Assign a value to radius
		radius = 123.4567;
		
		// Compute its area and its perimeter
		area = PI * radius * radius;
		perimeter = 2.0 * PI * radius;
		
		// Display the results
		System.out.print ("Given radius = ");
		System.out.println (radius);
		
		System.out.print ("Area = ");
		System.out.println (area);
		
		System.out.print ("Perimeter = ");
		System.out.println (perimeter);
	}
}

File: CSIT111/Lecture_2/Lecture_Labs/Lecturer_Examples/Circle_2a.py
class Circle_2a:
    def main(self):
        
        # Declare a constant pi
        PI = 3.14159256
        
        # Assign a value to radius
        radius = 123.4567
        
        # Compute its area and its perimeter
        area = PI * radius * radius
        perimeter = 2.0 * PI * radius
        
        # Display the results
        print("Given radius = " + str(radius))
        print("Area is " + str(area))
        print("Perimeter is " + str(perimeter))
        
circle_2a = Circle_2a
circle_2a().main()